### ISSUE TITLE
digi.no: ads

### ISSUE THREAD

**Issue by Schanzation** [uBlockOrigin/uAssets#23510]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.digi.no/artikler/denne-android-skadevaren-kan-stjele-bankdata-og-ta-over-mobilen-din/546361`

### Category

ads

### Description

This has a lot of ads. I tried different browsers and disabling all other extensions than ublock. The problem still persists.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/19490515/99ff2f75-b847-47f3-b910-3b7ef5e2f2a8)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.57.2
Firefox: 125
filterset (summary):
 network: 179493
 cosmetic: 181506
 scriptlet: 46525
 html: 2043
listset (total-discarded, last-updated):
 added:
  https://raw.githubusercontent.com/DandelionSprout/adfilt/master/LegitimateURLShortener.txt: 2624-0, now
  adguard-generic: 81295-3932, now
  adguard-spyware-url: 1507-235, now
  block-lan: 66-0, now
  curben-phishing: 14270-1, now
  fanboy-cookiemonster: 50981-186, now
  fanboy-social: 15825-13, now
  ublock-annoyances: 6900-106, now
  NOR-0: 3870-188, now
  adguard-popup-overlays: 27236-352, now
  adguard-widgets: 2490-12, now
  easylist-chat: 191-16, now
  [2 lists not shown]: [too many]
 default:
  user-filters: 18-0, never
  ublock-filters: 37895-109, now
  ublock-badware: 8461-0, now
  ublock-privacy: 876-52, now
  ublock-unbreak: 2322-2, now
  ublock-quick-fixes: 183-21, now
  easylist: 86905-1642, now
  easyprivacy: 50830-160, now
  urlhaus-1: 12815-0, now
  plowe-0: 3735-1170, now
filterset (user): [array of 18 redacted]
trustedset:
 added: [array of 11 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1042 ms (selfie)
 maxAssetCacheWait: 415 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 14
 network:
  digi.no: 5
  aticdn.net: 2
  cxense.com: 2
  doubleclick.net: 1
  licdn.com: 1
  linkedin.com: 1
  piano.io: 1
  tumedia.no: 1
 extended:
  ##.ad-label
  ##.mx-auto > div:empty:first-of-type
  ##div[class*=" clear"]
  ##footer:style(margin: 0 !important)
  ##+js(abort-on-property-write, __AB__)
  ##+js(remove-attr, class, .dfp-loaded)
```
</details>

**Comment by THEtomaso** [comment]
Not so bad, this time around.
`digi.no,tu.no##[id^="netBoard"]` seems to take care of all the new ones.

**Comment by THEtomaso** [comment]
> the recentmost proper suggestions I can find were 4 weeks ago.

Yeah, but you've consistently ended up with rules that's been different from the ones that I've suggested.
Don't get me wrong.. In most cases; I think your rules have been far "better", since I'm fully aware that mine often tend to be a tad "harsh".
Anyway, the reason why my rules still works, is probably simply because they've gone under the webmasters radar. :)

**Comment by DandelionSprout** [comment]
Oh, and while it's not entirely too late for me to point this out:

Their 2-site network (`digi.no` and `tu.no`) have all but bested me for months now, so I'd be significantly grateful if uAssets and potentially also Team AdGuard gave it some shots to fix any issues with those sites. Would've lifted a whole lot of weight off my shoulders, I must admit.
